src/index.html

```diff
@@ -22,6 +22,394 @@
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="review-text">Reviews: 5</p>
+        </div>
+        <div class="container__product-price">
+          <p class="price-text">Price:</p>
+          <span class="cost">$2,199</span>
+        </div>
+        <a
+          data-qa="card-hover"
+          href="#"
+          class="container__btn"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="container"
+        data-qa="card"
+      >
+        <img
+          class="container__image"
+          src="images/imac.jpeg"
+          alt="Apple Device(imac)"
+        />
+        <div class="container__product-info">
+          <h3 class="product-info-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="product-info-text">Product code: 195434</p>
+        </div>
+        <div class="container__stars-reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="review-text">Reviews: 5</p>
+        </div>
+        <div class="container__product-price">
+          <p class="price-text">Price:</p>
+          <span class="cost">$2,199</span>
+        </div>
+        <a
+          data-qa="card-hover"
+          href="#"
+          class="container__btn"
+        >
+          BUY
+        </a>
+      </div>
+
+      <div
+        class="container"
+        data-qa="card"
+      >
+        <img
+          class="container__image"
+          src="images/imac.jpeg"
+          alt="Apple Device(imac)"
+        />
+        <div class="container__product-info">
+          <h3 class="product-info-title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h3>
+          <p class="product-info-text">Product code: 195434</p>
+        </div>
+        <div class="container__stars-reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="review-text">Reviews: 5</p>
+        </div>
+        <div class="container__product-price">
+          <p class="price-text">Price:</p>
+          <span class="cost">$2,199</span>
+        </div>
+        <a
+          data-qa="card-hover"
+          href="#"
+          class="container__btn"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
The repeated blocks for products should be generated dynamically if possible, using a template and data source, to avoid code duplication and allow easier maintenance. Consider using a templating engine or JavaScript to render these blocks.